src/Search/README_SEARCH.md

```diff
@@ -66,32 +66,30 @@ $client = new SearchClient(HASHID, API_KEY);
 ### Quick & Dirty
 
 ```php
-require_once('path/to/php-doofinder/autoload.php');
-
-define('HASHID', '6a9gbc4dcx735x123b1e0198gf92e6e9');
-define('API_KEY', 'eu1-384fdag73c7ff0a59g589xf9f4083bxb9727f9c3')
-
-// Set hashid and API Key
-$client = new \Doofinder\Search\Client(HASHID, API_KEY);
-
-// You can specify filters
-$client->setFilter('brand', array('nike', 'converse')); // brand must be 'nike' or 'converse' AND ...
-$client->setFilter('color', array('red', 'blue'));      // ... color must be 'red' or 'blue' AND ...
-$client->setFilter('price', array('from'=>33.2));       // ... price must be upper than 33.2
-
-// You can also use more specific methods for that.
-$client->addTerm('brand', 'adidas');    // add 'adidas' to the 'brand' filter
-$client->removeTerm('brand', 'nike');   // remove 'nike' from the 'brand' filter
-$client->setRange('price', null, 99.9); // add an upper limit to the price
-
-// Feeling adventurous? sort!
-
-$client->addSort('price', 'desc'); // sort by price (descending)...
-$client->addSort('title', 'asc');  // ... and then by title (ascending)
-
-// Do the query, specify the page if you want.
-// 'page' = 1. optional . 'transformer' = 'dflayer'. optional.
-$results = $client->query('test query', 1, array('transformer' => 'dflayer'));
+require_once("path/to/php-doofinder/autoload.php");
+
+define("HASHID", "6a9gbc4dcx735x123b1e0198gf92e6e9");
+define("SERVER", "eu1-search.doofinder.com");
+define("API_KEY", "384fdag73c7ff0a59g589xf9f4083bxb9727f9c3")
+
+// Set server and API Key
+$client = new \Doofinder\Search\Client(SERVER, API_KEY);
+
+$searchParams = [
+    "query" => "sneakers",
+    "filter" => [
+        "brand" => ["nike", "converse"],
+        "color" => ["red", "blue"],
+        "price" => ["from" => 33.2, "to" => 99]
+    ],
+    "sort" => [
+        ["price" => "desc"],
+        ["title" => "asc"]
+    ],
+    "transformer" => "basic"
+];
+
```

Comment:
`"hashid"` param is missing here, and it's mandatory, right??

Reply:
Right, thanks!